sources/install/package_web.sh

```diff
@@ -894,6 +894,19 @@ function install_postman() {
     add-history postman
     add-test-command "which postman"
     add-to-list "postman,https://www.postman.com/,API platform for testing APIs"
+
+function install_zap() {
+    colorecho "Installing ZAP"
+    zap_version=$(curl -s "https://www.zaproxy.org/download/" | grep -E -o "ZAP [0-9]+(\.[0-9]+)+" | head -1 | cut -d' ' -f2)
+    wget "https://github.com/zaproxy/zaproxy/releases/download/v${zap_version}/ZAP_${zap_version}_Linux.tar.gz" -O /tmp/zap.tar.gz
+    tar -xf /tmp/zap.tar.gz -C /tmp/
+    mv /tmp/ZAP_${zap_version} /opt/tools/ZAP
+    rm -rf /tmp/ZAP_${zap_version}
+    rm /tmp/zap.tar.gz
+    add-aliases zap
+    add-history zap
+    add-test-command "which zap"
```

Comment:
Not a big fan of that test command to be honest, is there any way to test for the install?

Reply:
The problem is that there's no command available through the CLI. Any command start the GUI

Reply:
I'm not sure which command to add, it's the same problem as burpsuite for example.